EosPayload/lib/base_drivers/driver_base.py

```diff
@@ -1,8 +1,8 @@
-from abc import ABC, abstractmethod
+import threading
+from abc import ABC
```

Comment:
Well, it's an abstract class in the sense that you shouldn't be instantiating a DriverBase. It might be worth making setup and cleanup be abstract methods? Also a main method once we move to a single main method that spawns various threads or something

Reply:
Yeah, you're right, but I still kinda hate it